internal/service/model_errors.go

```diff
@@ -24,6 +24,7 @@ import (
 
 type ErrorAlerting struct {
 	PagerDuty *PagerDutyAlerting
+	Slack     *SlackAlerting
 }
```

Comment:
Can we update the docs to include Slack's configuration options?

https://github.com/moov-io/achgateway/blob/master/docs/CONFIGURATION.md#error-alerting